packages/flutter/lib/src/services/text_input.dart

```diff
@@ -837,12 +836,12 @@ class TextInput {
         // TODO(flutterweb): what makes sense here?
         return true;
       }
-      if (Platform.isIOS) {
+      if (defaultTargetPlatform == TargetPlatform.iOS) {
```

Comment:
Looks like we have a lot of tests that rely on this being false for both :(

Reply:
I think these asserts were essentially never tested. Maybe we should remove them?

Reply:
The tester would resolve this to the host platform right?

We should fix the test. I believe the asserts work as intended even if they're not correctly tested. And this should be a `switch`.

Reply:
this is definitely not default Target Platform fodder. Changing the UI style to Android doesn't magically make it support iOS actions.